src/data/__tests__/calculate.test.ts

```diff
@@ -132,28 +134,33 @@ describe('calculate', () => {
 
     const response = calcValue(data)
     // average * 2 people * outdoor * 1 hr * their mask * your mask
-    expect(response).toBe(((((PREVALENCE * 2) / 20) * 0.06) / 4 / 1) * 1e6)
+    expect(response).toBeCloseTo(
+      ((((PREVALENCE * 2) / 20) * 0.06 * B117_CONTAGIOUSNESS_ADJUSTMENT) /
+        4 /
+        1) *
+        1e6,
+    )
   })
 
   it.each`
     scenario                                                        | result
-    ${'Outdoor masked hangout with 2 people'}                       | ${9}
-    ${'Indoor unmasked hangout with 2 people'}                      | ${720}
-    ${'Car ride with 1 person for 15 mins'}                         | ${90}
-    ${'One-night stand with a random person'}                       | ${2880}
-    ${'Live-in partner who has no indoor interactions besides you'} | ${21.6}
-    ${'Grocery store for 60 minutes (average # of shoppers)'}       | ${90}
-    ${'Grocery store for 60 minutes (few other shoppers)'}          | ${45}
-    ${'Plane ride (full flight)'}                                   | ${369}
-    ${'Plane ride (middle seat empty)'}                             | ${180}
-    ${'Eating in restaurant, outdoors'}                             | ${202.5}
-    ${'Eating in restaurant, indoors'}                              | ${4050}
-    ${'Going to bar'}                                               | ${27000}
-    ${'Large outdoor party: masked with 3 feet between people'}     | ${1080}
-    ${'Small indoor party: unmasked with 25 people'}                | ${27000}
-    ${'Outdoor, masked hangout with person who has COVID'}          | ${750}
-    ${'Indoor, unmasked hangout with person who has COVID'}         | ${60000}
-    ${'Voting in-person'}                                           | ${3}
+    ${'Outdoor masked hangout with 2 people'}                       | ${9 * B117_CONTAGIOUSNESS_ADJUSTMENT}
+    ${'Indoor unmasked hangout with 2 people'}                      | ${720 * B117_CONTAGIOUSNESS_ADJUSTMENT}
+    ${'Car ride with 1 person for 15 mins'}                         | ${90 * B117_CONTAGIOUSNESS_ADJUSTMENT}
+    ${'One-night stand with a random person'}                       | ${2880 * B117_CONTAGIOUSNESS_ADJUSTMENT}
+    ${'Live-in partner who has no indoor interactions besides you'} | ${21.6 * B117_CONTAGIOUSNESS_ADJUSTMENT ** 2}
```

Comment:
I don't totally understand why it's to the power of 2 instead of x2. But I believe you that it's accurate.

Reply:
Live-in partner is calculated based on 1 hour of grocery shopping. Grocery shopping is 1.5x more dangerous, so Live-in partner is 1.5x more points. Then you have an interaction with live-in partner, which has 1.5x more activity risk because of b117.

old value = old person risk * old activity risk
new value = new person risk * new activity risk = (old person risk * 1.5) * (old activity risk * 1.5) = old value * 1.5 * 1.5

Reply:
Gotcha. could you add a comment explaining this since it's no totally obvious upon reading how the chain effects works.

Reply:
No longer applicable.

But... in general everything in the calculator is multiplicative. The only thing that is additive is # of people.